backend/requirements.txt

```diff
@@ -38,9 +38,9 @@ jsonpatch==1.33
 jsonpointer==3.0.0
 langchain==0.3.1
 langchain-community==0.3.1
-langchain-core==0.3.6
+langchain-core==0.3.25
 langchain-experimental==0.3.2
-langchain-ollama==0.2.0
+langchain-ollama==0.2.1
```

Comment:
Updated langchain-ollama from 0.2.0 to 0.2.1, which may include new features or bug fixes.